src/objects/ad-study.js

```diff
@@ -80,6 +80,12 @@ export default class AdStudy extends AbstractCrudObject {
     );
   }
 
+  deleteAdStudy(params): AbstractObject {
```

Comment:
@michaelwclark Functionality wise this worked for me. Could we rename the function from `deleteAdStudy` to `delete` to be more inline with the deletion functions used for other objects?

Reply:
@CarlousF this has been pushed